packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -265,29 +283,45 @@ export const prepareLocalOrchestrationAccountKit = (
        */
       returnVoidWatcher: {
         /**
-         * @param {Record<unknown, unknown>[]} results
+         * @param {any} _result
```

Comment:
Please let's avoid `any` unless there's a strong reason to use it.
Use `unknown` instead; it's type-safe.